Makefile.PL

```diff
@@ -61,7 +61,7 @@ if ($ENV{EXTENDED_TESTING}) { # for CpanCover and masochists
     # Versions also need to be declared inside Read.pm !
     $wm{PREREQ_PM}{"Text::CSV_PP"}		= "1.97";
     $wm{PREREQ_PM}{"Text::CSV_XS"}		= "1.36";
-    $wm{PREREQ_PM}{"Spreadsheet::ReadSXC"}	= "0.20";
+    $wm{PREREQ_PM}{"Spreadsheet::ReadSXC"}	= "0.23";
```

Comment:
I generally do not **require** the most recent version: I have a minimum version I support because of functionality. I (of course) prefer the most recent stable/working version, and than I disable the tests for the new(est) functionality for versions older than that. This way the module does not require the newest version of the parser and will still work on the module it used to work on.